### PR TITLE
[SC] SmartContractWallet History

### PR DESCRIPTION
**Pull request by codingupastorm**

### CODE REVIEW

File: src/Stratis.StratisSmartContractsD/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Stratis.StratisSmartContractsD": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "-connect=52.189.213.175"
```
Comment (rowandh) [inline src/Stratis.StratisSmartContractsD/Properties/launchSettings.json]: Remove

File: src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractAccountHistoryModel.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
```
Comment (rowandh) [inline src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractAccountHistoryModel.cs]: Looks this this file and some others are duplicated between the two wallet projects. Was this intentional?

File: src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractWalletHistoryModel.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public class ContractWalletHistoryModel
+    {
+        public ContractWalletHistoryModel()
+        {
+            this.AccountsHistoryModel = new List<ContractAccountHistoryModel>();
+        }
+
+        [JsonProperty(PropertyName = "history")]
+        public ICollection<ContractAccountHistoryModel> AccountsHistoryModel { get; set; }
```
Comment (rowandh) [inline src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractWalletHistoryModel.cs]: Each wallet history request specifies the account name so I assume this should only ever have one item? Feels like unnecessary nesting if so.

File: src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractAccountHistoryModel.cs (hunk 2 above)
Comment (codingupastorm) [reply src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractAccountHistoryModel.cs]: They're nearly identical but with a different enum I believe is the only change. This little change forced all the concrete classes to change

**Pull request closed without merging**